peps/pep-0728.rst

```diff
@@ -384,14 +396,14 @@ For example::
 
```

Comment:
I know this is preexisting, but there's a typo three lines above (misspelling of overridden).

(Sadly, I can't comment on the exact line - https://github.com/orgs/community/discussions/4452.)

Reply:
And another:
```console
❯ make spellcheck
...
peps/pep-0728.rst:393: overriden ==> overridden
peps/pep-0728.rst:706: othere ==> other
```

Reply:
Thanks, fixed both